packages/wallet/src/KeyManagement/util/mapHardwareSigningData.ts

```diff
@@ -0,0 +1,567 @@
+// Based on Nami implementation
+// https://github.com/Berry-Pool/nami-wallet/blob/main/src/api/util.js
```

Comment:
Link to specific commit

Reply:
Thanks @rhyslbw . Comment updated and e2e with ledger are green.